DQM/HLTEvF/plugins/TriggerRatesMonitor.cc

```diff
@@ -178,9 +173,11 @@ void TriggerRatesMonitor::dqmBeginRun(edm::Run const &run,
   edm::EDConsumerBase::Labels labels;
   labelsForToken(m_hlt_results, labels);
   if (histograms.hltConfig.init(run, setup, labels.process, changed)) {
-    histograms.hltIndices.resize(histograms.hltConfig.size());
+    // number of trigger paths in labels.process
+    auto const nTriggers = histograms.hltConfig.size();
+    histograms.hltIndices.resize(nTriggers);
 
-    unsigned int datasets = histograms.hltConfig.datasetNames().size();
+    unsigned int const datasets = histograms.hltConfig.datasetNames().size();
```

Comment:
Perhaps, in analogy to `nTriggers` here above, you can profit of this PR and rename this size as `nDatasets`: as such you'd also aviod giving tha same name to a (vector) class member and to this size, which can be confusing

Reply:
Thanks for the suggestion. This is now in 94344d9. Diff of the latest changes:
https://github.com/cms-sw/cmssw/compare/cc9a15215b3aaebe6bb5ea036fa894e0bcf77972..94344d9f41472e1fe769a931001fdf20d501c1a0